packages/lodestar/src/network/subnetsService.ts

```diff
@@ -34,10 +42,18 @@ export interface IAttestationServiceModules {
   metadata: MetadataController;
 }
 
+export function getAttnetsService(modules: IAttestationServiceModules): SubnetsService {
+  return new SubnetsService(modules, GossipType.beacon_attestation, "attnets");
+}
+
+export function getSyncnetsService(modules: IAttestationServiceModules): SubnetsService {
+  return new SubnetsService(modules, GossipType.sync_committee, "syncnets");
+}
+
 /**
  * Manage random (long lived) subnets and committee (short lived) subnets.
  */
-export class AttestationService implements IAttestationService {
```

Comment:
maybe we don't need to change this class as it's specific for attestation only

Reply:
I've generalized it for attnets and syncnets